Wireframe/css/style.css

```diff
@@ -0,0 +1,177 @@
+body{
+    margin-right: 10%;
+    margin-left: 10%;    
+    text-align: center;
+    background-color: #1e1c1cea;
+    font-size: 120%;
+}
+
+
+header{
+    text-align: center;
+    border-bottom: solid 5px #ffffff;
+}
+
+
+h1{
+    font-size: 45px;
+    font-weight: bold;
+    color: #ff7b00;
+}
+
+
+h2{
+    text-align: center;
+    font-size: 30px;
+    color: #ff7b00;
+    font-weight: bolder;
+}
+
+
+h3{
+    font-size: 26px;
+    color: #ff7b00;
+}
+
+
+li{
+    padding-bottom: 15px;
+}
+
+
+/* Header */
+.pHeader{
```

Comment:
This is an interesting class - as it may not be required. In CSS, there's lots of different selectors we can create, for example `header p` (with a space between `header` and `p`) selects all `p` elements within a `header` element. This means that you won't need to create a separate class just for this, and it is easier for other developers to see what is going on.